### PR TITLE
DOCS-14189 issues in writing guidlines

### PR DESCRIPTION
**Pull request by davemungo**
[STAGING](https://docs-mongodbcom-staging.corp.mongodb.com/meta/docsworker-xlarge/DOCS-14189-issues-in-writing-guidlines/style-guide/)

[JIRA](https://jira.mongodb.org/browse/DOCS-14189)

### CODE REVIEW

**Comment by davemungo** [comment]
@jeff-allen-mongo  Tagging you as the style guide rep as per the wiki. Let me know if this you go to someone else.

File: source/style-guide/style/text-formatting.txt [hunk 1]
```diff
@@ -1195,31 +1197,6 @@ elements. The following style differences are highlighted:
           - Client authentication is provided through a REST interface
             by using the ``GET`` method.
 
-   New terms
```
Comment (jeff-allen-mongo) [inline source/style-guide/style/text-formatting.txt:1198]: [question]

Was this section deleted intentionally?

**Review by jeff-allen-mongo — COMMENTED** [review]
@davemungo thanks for the improvements here. I left a few comments.

**Review by ccho-mongodb — CHANGES_REQUESTED** [review]
Good eyes finding these corrections! I Identified a few issues and left a few suggestions and questions.

File: source/style-guide/style/text-formatting.txt [hunk 2]
```diff
@@ -535,12 +535,12 @@ elements. The following style differences are highlighted:
                :copyable: false
 
                The following example shows a basic configuration for
-               the FTP service, in a file in the ``/etc/xinetd.d
-               directory``.
+               the FTP service, in a file in the ``/etc/xinetd.d``
+               directory.
```
[the remark this answers is not in the document]
Comment (davemungo) [reply source/style-guide/style/text-formatting.txt]: Done

File: source/style-guide/style/text-formatting.txt (hunk 1 above)
Comment (davemungo) [reply source/style-guide/style/text-formatting.txt:1177]: The section is included, with a slight change in wording, under "Terms, new" starting around line 1551

**Review by davemungo — COMMENTED** [review]
@jeff-allen-mongo @ccho-mongodb  thanks for your feedback. This is ready for another pass

File: source/style-guide/style/capitalization.txt [hunk 3]
```diff
@@ -188,7 +189,7 @@ an incorrect capitalization:
    * - REST
          Representational State Transfer
      - rest
-         State of motionless or inactivity
+         State of motionlessness or inactivity
```
[the remark this answers is not in the document]
Comment (jeff-allen-mongo) [reply source/style-guide/style/capitalization.txt:191]: I think the concern with this table is that sometimes the first letter of the expansions are capitalized, and sometimes they're not.

For example, `Kbps` exapnds to `kilobits per second` (all lower case) whereas `May` "expands" to `Fifth month of the year` (with a capital F).

My suggestion would be to revert "REpresentational" to the original capitalization of "Representational" and capitalize the first letter of each expansion.

@ccho-mongodb let me know if I've misunderstood your feedback or you have a different suggestion.

[the remark this answers is not in the document]
Comment (davemungo) [reply source/style-guide/style/capitalization.txt:191]: Done

**Review by jeff-allen-mongo — APPROVED** [review]
LGTM

File: source/style-guide/style/capitalization.txt (hunk 3 above)
[the remark this answers is not in the document]
Comment (ccho-mongodb) [reply source/style-guide/style/capitalization.txt:191]: I think it's a really minor issue which is I why I mentioned no changes necessary.

Both expansions and definitions are represented in the table without clearly discernment of the case, so it seems capitalization of sentence fragments in a list could apply here. The focus seems to be on showing how the difference in capitalization of an abbreviated term can have a different meaning than a similarly-spelled term that isn't capitalized.

File: source/style-guide/style/punctuation.txt [hunk 4]
```diff
@@ -133,7 +133,7 @@ Manual of Style.
        Unlike the other alarms in this list you set the network check
        alarm variable upon network check creation.
 
-       To learn more, see Upgrading Ops Manager.
+       To learn more see Upgrading Ops Manager.
```
[the remark this answers is not in the document]
Comment (ccho-mongodb) [reply source/style-guide/style/punctuation.txt:136]: Question:

Isn't the comma in the original version correct in showing how to properly use the comma to separate the introductory phrase/clause?

File: source/style-guide/style/punctuation.txt [hunk 5]
```diff
@@ -347,7 +349,9 @@ Periods
 ~~~~~~~
 
 Use the following guidelines for periods. For basic period usage, see a
-grammar reference, `section 6.12, Periods <https://www.chicagomanualofstyle.org/book/ed17/part2/ch06/psec012.html>`__ in the *Chicago Manual of Style*.
+grammar reference, `section 6.12, Periods
+<https://www.chicagomanualofstyle.org/book/ed17/part2/ch06/psec012.html>`__
+in the *Chicago Manual of Style*.
```
Comment (ccho-mongodb) [inline source/style-guide/style/punctuation.txt:354]: Apologies if it wasn't part of the JIRA ticket task to fix all the instances of CMOS, but it looks like theres no link in the "Hyphens with prefixes" section which I spotted when searching for this link.

File: source/style-guide/style/text-formatting.txt [hunk 6]
```diff
@@ -122,7 +115,7 @@ elements. The following style differences are highlighted:
 
    Box names
      This covers check box, combo box, group box, list box, spin box,
-     text box, but not dialog box. Format using ``:guilabel:`` role.
+     and text box. Format them using ``:guilabel:`` role.
```
Comment (ccho-mongodb) [inline source/style-guide/style/text-formatting.txt:118]:
```suggestion
     and text box. Format them using the ``:guilabel:`` role.
```

File: source/style-guide/style/text-formatting.txt [hunk 7]
```diff
@@ -514,12 +507,12 @@ elements. The following style differences are highlighted:
                :copyable: false
 
                The following example shows a basic configuration for
-               the FTP service, in a file in the ``/etc/xinetd.d
-               directory``.
+               the FTP service, in a file in the ``/etc/xinetd.d/``
```
Comment (ccho-mongodb) [inline source/style-guide/style/text-formatting.txt:510]: Question:

Should the double backticks exist here? If I remember correctly, the double-backtick within a code block used to have some rendering behavior associated with it, but I think that was removed a while back. If uncertain or if there's some guidance around representing monospace paths this way, no changes necessary!

**Review by ccho-mongodb — APPROVED** [review]
LGTM. Added a couple more suggestions and questions.

**Review by krollins-mdb — APPROVED** [review]
LGTM. Thanks for all the work you put into this!

File: source/style-guide/style/punctuation.txt (hunk 4 above)
[the remark this answers is not in the document]
Comment (davemungo) [reply source/style-guide/style/punctuation.txt]: Done

File: source/style-guide/style/punctuation.txt (hunk 5 above)
Comment (davemungo) [reply source/style-guide/style/punctuation.txt:355]: Done. 

The other links are paywalled. Opened https://jira.mongodb.org/browse/DOCS-15855 to replace them.

File: source/style-guide/style/text-formatting.txt (hunk 6 above)
Comment (davemungo) [reply source/style-guide/style/text-formatting.txt]: Done

File: source/style-guide/style/text-formatting.txt (hunk 7 above)
Comment (davemungo) [reply source/style-guide/style/text-formatting.txt:510]: I'm not aware of any prohibitions, but the section renders properly. Leaving as is for now.

**Review by davemungo — COMMENTED** [review]
@ccho-mongodb  I've updated the doc with the latest comments.

**Review by corryroot — APPROVED** [review]
LGTM

**Pull request merged**